### Statistics for Measurement Data Example 3.1
We measure the fusion heat with a particular method and find for the empirical mean value of the $ n= 13 $ measurements $ 80.02 $.
Let us assume that we know from previous studies that the standard deviation of our measuring instrument is $\sigma_X = 0.01$. Is it plausible that the fusion heat is exactly $80.00{cal/gram}$? We perform a $z$-test for this purpose: 

1. **Model:** $X_i$ is a continuous measurement variable; 
\begin{equation*} 
  X_1,\ldots ,X_n\ i.i.d. \sim\mathcal{N}(\mu,\sigma_X^2), \quad \sigma_X=0.01\textrm{ known}, \quad n=13
\end{equation*} 

2. **Null hypothesis:**
\begin{equation*} 
  H_0:\quad \mu = \mu_0 = 80.00 
\end{equation*} 

**Alternative:**
\begin{equation*} 
  H_A:\quad\mu \neq \mu_0 
\end{equation*} 

3. **Test statistic:**
\begin{equation*} 
Z 
= \frac{\sqrt{n}(\overline{X}_n-\mu_0)}{\sigma_X} 
\end{equation*} 
Null distribution (assuming $H_0$ is true): 
\begin{equation*} 
  Z \sim\mathcal{N}(0,1) 
\end{equation*} 

4. **Significance level:**
\begin{equation*} 
  \alpha = 0.05 
\end{equation*} 

5. **Rejection region for the test statistic:**
\begin{equation*} 
K 
= (-\infty, z_{\frac{\alpha}{2}}] \cup [z_{1-\frac{\alpha}{2}}, \infty) \quad \text{with}\quad H_A:\ \mu \neq \mu_0, 
\end{equation*} 

Given $ \alpha=0.05 $, **Python** yields the following 2.5% quantile of the standard normal distribution.

In [1]:
from scipy.stats import norm

q_25 = norm.ppf(0.025)
print(q_25)

-1.9599639845400545


\begin{equation*} 
z_{\frac{\alpha}{2}} =\Phi^{-1}\left (\frac{\alpha}{2}\right )  = \Phi^{-1}(0.025)  = -1.96 
\end{equation*} 
The following rejection region for the test statistic results
\begin{equation*} 
K = (-\infty, -1.96] \cup [1.96, \infty) 
\end{equation*} 

6. **Test decision:** 

On the basis of the $n=13$ measurements we find for the empirical mean  
\begin{equation*} 
\overline{x}_n =80.02 
\end{equation*} 
Hence the value for the test statistic is 
\begin{equation*} 
  z =\frac{\sqrt{13}(80.02 - 80.00)}{0.01} = 7.211 
\end{equation*} 
Since the observed value of the test statistic falls into the rejection region, we reject the null hypothesis at the 5% significance level.



1. **Model:** $X_i$ is a continuous measured variable; 
\begin{equation*}
  X_1,\ldots ,X_n\ iid \sim\mathcal{N}(\mu,\sigma_X^2), \quad \sigma_X=0.01\textrm{ known}, n=13
\end{equation*} 

2. **Null hypothesis:**
\begin{equation*}
  H_0:\quad \mu = \mu_0 = 80.00 
\end{equation*} 


**Alternative:**
\begin{equation*}
  H_A:\quad\mu \neq \mu_0 
\end{equation*} 

3. **Test statistic:**

The mean value of the measurements
\begin{equation*}
  T:\quad \overline{X}_{n} 
\end{equation*} 

**Null distribution (assuming $H_0$ is true):**
\begin{equation*}
  T \sim\mathcal{N}\left (\mu_{0},\dfrac{\sigma_{X}^{2}}{n}\right ) =\mathcal{N}\left (80,\dfrac{0.01^{2}}{13}\right ) 
\end{equation*} 

4. **Significance level:**
\begin{equation*}
  \alpha  = 0.05 
\end{equation*} 


5. **Rejection region for the test statistic:**
\begin{equation*} 
K 
= (-\infty, c_{u}] \cup [c_{o}, \infty) \quad \text{with}\quad H_A:\ \mu \neq \mu_0, 
\end{equation*} 

In [3]:
import numpy as np

rej_reg_5 = norm.interval(alpha=0.95, loc=80, scale=0.01/np.sqrt(13))
print(rej_reg_5)

(79.9945640379659, 80.0054359620341)


In this way, we obtain the rejection region for the test statistic:
\begin{equation*}
K = (-\infty, 79.99] \cup [80.01, \infty) 
\end{equation*} 


6. **Test decision:**

On the basis of the $n=13$ measurements, we calculate 
\begin{equation*}
\overline{x}_n =80.02 
\end{equation*} 
The observed value falls into the rejection region of the test statistic. Hence, the null hypothesis at the 5% significance level is rejected. 